kuma/static/styles/components/content.scss

```diff
@@ -81,10 +81,57 @@ We cannot put these styles in the base/elements/ files because they don't apply
     sectioning
     ====================================================================== */
 
-    h2 {
+    /* direct decendants h2s get styled like section seperators
+       so do things we think are h2s added by macros */
+    article[id=wikiArticle] > h2,
+    article[id=wikiArticle] > div:not([class]) > h2 {
+        @include heading-2-section();
+    }
+
+    /*  remove top spacing if h2 is effectivly first element in article */
+    article[id=wikiArticle] > h2:first-child,
+    article[id=wikiArticle] > div:first-child:empty + h2,
+    article[id=wikiArticle] > p:first-child:empty + h2,
+    .Quick_links + h2 {
+        margin-top: 0;
+        padding-top: 0;
+
+        &:before {
+            display: none;
+        }
+    }
+
+    /* fix spacing and line if h2 directly follows a hr, summary, or blockquote */
+    article[id=wikiArticle] > hr + h2,
+    article[id=wikiArticle] > .summary + h2,
+    article[id=wikiArticle] > blockquote + h2 {
+        margin-top: ($grid-spacing + $section-border-width) * -1;
+        padding-top: $grid-spacing * 2 + $section-border-width;
+        background-color: #fff;
+
+        &:before {
+            top: 0;
+        }
+    }
+
+    h3,
+    h5 {
         @extend %highlight;
     }
 
+    h3,
+    h3 .highlight-span {
+        @include bidi-value(padding, 0 4px 0 $grid-spacing, 0 $grid-spacing 0 4px);
```

Comment:
This turns into `padding: 0 4px 0 40px;` - is that intentional or should it be `padding: 0 4px 0 4px;`?

<img width="222" alt="screen shot 2017-11-08 at 14 27 13" src="https://user-images.githubusercontent.com/10350960/32549130-f45daf88-c490-11e7-847c-a5a70d6c6aa7.png">

Reply:
That's intentional. You can see the indentation on headings on moz org, like here: https://www.mozilla.org/en-US/firefox/